### PR TITLE
lowercase blueprints

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/developer-experience/issues/389 -- lowercasing a few cases of blueprints.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To launch Web Modeler, follow the steps below:
 2. Select **Create new project** to create a new project and store diagrams.
    ![web modeler empty home](img/web-modeler-new-user-home.png)
 3. Name your diagram. You can go back and change the name any time by clicking on the project name and **Edit name**.
-4. Select **Browse blueprints** to view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagrams.
+4. Select **Browse blueprints** to view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.
```
Comment (akeller) [inline versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md:17]: Is this inconsistently cased in the product? 

Browse blueprints vs. Use Blueprints?

Comment (christinaausley) [reply versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md:17]: It is, yes. I can likely launch a PR in the product to adjust!

**Review by mesellings — APPROVED** [review]
Lgtm 👍 🚀

File: versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md (hunk 1 above)
Comment (YanaSegal) [reply versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md:17]: The confusion is from the text, not the product. The user clicks "Browse blueprints" to open the blueprints dialog. Once the user finds a blueprint they want to use, they need to click on "Use blueprint," which will download the blueprint into the project (and open it in the diagram screen).

Comment (christinaausley) [reply versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md:17]: Here is what I currently see in the product:

Screenshot 2024-09-26 at 9 19 07 AM
Screenshot 2024-09-26 at 9 19 20 AM

Comment (christinaausley) [reply versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md:17]: Are you saying that you would prefer to see the text rewritten as:

Select **Browse blueprints** to open the blueprints dialog and view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**, which will download the blueprint into the project and open it in the diagram screen. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.

If so, I'll open a separate issue to revisit this.

Comment (YanaSegal) [reply versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md:17]: Yes excatly. Thank you :)

**Review by YanaSegal — APPROVED** [review]

**Pull request merged**